packages/stream_chat_flutter/lib/src/message_search_list_view.dart

```diff
@@ -144,61 +144,69 @@ class _MessageSearchListViewState extends State<MessageSearchListView> {
+                      minHeight: viewportConstraints.maxHeight,
+                    ),
+                    child: const Center(
+                      child: Text('There are no messages currently'),
                     ),
                   ),
-          errorBuilder: widget.errorBuilder ??
-              (BuildContext context, dynamic error) {
-                if (error is Error) {
-                  print(error.stackTrace);
-                }
-                return InfoTile(
-                  showMessage: widget.showErrorTile,
-                  tileAnchor: Alignment.topCenter,
-                  childAnchor: Alignment.topCenter,
-                  message: 'An error occurred.',
-                  child: Container(),
-                );
-              },
-          loadingBuilder: widget.loadingBuilder ??
-              (context) => LayoutBuilder(
-                    builder: (context, viewportConstraints) =>
-                        SingleChildScrollView(
-                      physics: const AlwaysScrollableScrollPhysics(),
-                      child: ConstrainedBox(
-                        constraints: BoxConstraints(
-                          minHeight: viewportConstraints.maxHeight,
-                        ),
-                        child: const Center(
-                          child: CircularProgressIndicator(),
-                        ),
-                      ),
+                ),
+              ),
+      errorBuilder: widget.errorBuilder ??
+          (BuildContext context, dynamic error) {
+            if (error is Error) {
+              print(error.stackTrace);
+            }
+            return InfoTile(
+              showMessage: widget.showErrorTile,
+              tileAnchor: Alignment.topCenter,
+              childAnchor: Alignment.topCenter,
+              message: 'An error occurred.',
+              child: Container(),
+            );
+          },
+      loadingBuilder: widget.loadingBuilder ??
+          (context) => LayoutBuilder(
+                builder: (context, viewportConstraints) =>
+                    SingleChildScrollView(
+                  physics: const AlwaysScrollableScrollPhysics(),
+                  child: ConstrainedBox(
+                    constraints: BoxConstraints(
+                      minHeight: viewportConstraints.maxHeight,
+                    ),
+                    child: const Center(
+                      child: CircularProgressIndicator(),
                     ),
                   ),
-          childBuilder: widget.childBuilder ?? _buildListView,
-        ),
-      );
+                ),
+              ),
+      childBuilder: widget.childBuilder ?? _buildListView,
+    );
+
+    final backgroundColor =
+        MessageSearchListViewTheme.of(context).backgroundColor;
+    return backgroundColor != null
+        ? ColoredBox(
+            color: backgroundColor,
+            child: messageSearchListCore,
+          )
+        : messageSearchListCore;
+  }
```

Comment:
LGTM, but I would have written this similar to message-list-view for consistency and easier to read.